l3kernel/l3basics.dtx

```diff
@@ -1197,9 +1197,7 @@
 %     \cs{cs_if_free:NTF} \meta{control sequence} \Arg{true code} \Arg{false code}
 %   \end{syntax}
 %   Tests whether the \meta{control sequence} is currently free to
-%   be defined. This test is \texttt{false} if the
-%   \meta{control sequence} currently exists (as defined by
-%   \cs{cs_if_exist:NTF}).
+%   be defined, meaning the opposite of the above \cs{cs_if_exist:NTF}.
```

Comment:
I can't really see that this is clearer than the original text, on the contrary. I would suggest not to make this change.

Reply:
The meaning of "free to be defined" is unclear. For example, temporary variables may be free to be defined despite they still contain material.
According to some comment in the implementation, `\cs_if_free:NTF` is the "logical reversal" of `\cs_if_exist:NTF`, this fact does not appear clearly in the actual documentation.

Reply:
sorry opinions differ then, it is clearly defined in the  (...) in my opinion, while  "meaning the opposite" leaves more question marks.

Reply:
if you want to strengthen the statement then I would do

This test is \texttt{false} if the \meta{control sequence} currently exists as defined by
\cs{cs_if_exist:NTF}, i.e., is the negation of that conditional.

Reply:
Why not simply say
"""
This test is the negation of \cs{cs_if_exist:NTF}.
"""

Reply:
also fine